distributed/cli/tests/test_dask_scheduler.py

```diff
@@ -315,6 +315,20 @@ def test_preload_remote_module(loop, tmp_path):
                 )
 
 
+def test_preload_config(loop):
+    # Ensure dask-scheduler pulls the preload from the Dask config if
+    # not specified via a command line option
+    with tmpfile() as fn:
+        env = os.environ.copy()
+        env["DASK_DISTRIBUTED__SCHEDULER__PRELOAD"] = PRELOAD_TEXT
```

Comment:
@crusaderky it looks like we need to include the existing environment values for Python to properly run on Windows (xref https://github.com/dask/distributed/pull/4511#issuecomment-924820855)